index.js

```diff
@@ -30,8 +36,12 @@ module.exports = (stack, options) => {
 		})
 		.filter(line => line.trim() !== '')
 		.map(line => {
+			if (basePathRe) {
+				line = line.replace(basePathRe, (_, p1) => p1);
```

Comment:
I think a string in the replacement parameter would be more efficient than a method.

Reply:
Replaced with `'$1'`, thought I prefer the function form personally.